.github/workflows/validate-other-market-data.yml

```diff
@@ -35,8 +36,8 @@ jobs:
       - name: Checkout v4-chain repo
         uses: actions/checkout@v3
         with:
-          repository: 'dydxprotocol/v4-chain'
-          ref: '7604f974a699efa8d15a8a70bfc85d8b5b2e2d58'
+          repository: 'skip-mev/v4-chain'
```

Comment:
this tag is pointing to a skip fork temporarily, pending [this](https://github.com/dydxprotocol/v4-chain/pull/1452) PR